composer.json

```diff
@@ -9,5 +9,7 @@
         "irc": "irc://irc.freenode.org/drupal-og",
         "source": "https://cgit.drupalcode.org/og"
     },
-    "minimum-stability": "dev"
+    "require-dev": {
+        "drupal/coder": "^8.2"
```

Comment:
This should follow stable releases, meaning it will be "fixed" for 3-4 months in between Coder releases. When a new release is out we will be instantly alerted :dancer: 

I prefer this over having to manually check if there is a new Coder release out every day, but it's up to you if you want to fix it to a specific version.